css/style.css

```diff
@@ -303,61 +326,116 @@ hr {
   font-family: "Roboto", serif;
   font-style: normal;
   font-weight: 350px;
-  font-size: 32px;
+  font-size: 24px;
   line-height: 44px;
   text-align: left;
   color: #fff;
   padding: 10px 0;
-  margin: 20px 25px;
-  margin-right: 24px;
+  margin: 20px 10px;
+  margin-left: 800px;
+  margin-top: -320px;
+}
+
+.title-post-left h2 {
+  font-family: 'Roboto', serif;
+  font-style: normal;
+  font-weight: 350;
+  font-size: 24px;
+  line-height: 44px;
+  display: flex;
+  align-items: left;
+  color: #fff;
+  margin-top: -300px;
+  margin-left: 160px;
+  padding: 10px 0;
 }
 
 .paragraph2 {
-  font-family: "Roboto", serif;
   font-weight: 200;
-  font-size: 20px;
+  font-size: 12px;
   line-height: 28px;
   color: #dcdce8;
 
   /* padding: 40px;
 padding-left: 29px; */
   text-align: left;
-  margin: 10px 24px;
+  margin: 10px 20px;
+  margin-left: 800px;
+  margin-top: -240px;
+}
+
+.paragraph-left {
+  font-family: 'Roboto', serif;
+  font-weight: 200;
+  font-size: 12px;
+  line-height: 28px;
+  color: #dcdce8;
+
+  /* padding: 40px;
+padding-left: 29px; */
+  margin-top: -220px;
+  margin-right: 600px;
 }
 
 .programming-languages {
   display: flex;
-  align-items: center;
-  text-align: center;
-  justify-content: center;
-  padding: 5px;
-  padding-left: 18px;
-  color: #f5f5ff;
-  column-gap: 1.5rem;
-  flex-direction: row;
+
+  /* align-items: center; */
+
+  /* justify-content: center; */
+
+  /* padding: 3rem; */
+  gap: 0.3rem;
+  margin-top: 15px;
+}
+
+form li + li {
+  margin-top: 1em;
+}
+
+.programming-languages li {
+  padding: 0.7rem;
 }
 
 .programming-languages li a {
   color: #fff;
 }
 
+.social-icons li {
+  margin-right: 10px;
+}
+
+.programming-languages li:not(:last-child) {
+  border-right: 1px solid #333;
+}
+
 .btn {
+  width: 120px;
+  height: 65px;
   color: #fff;
-  padding: 25px 25px;
+  padding: 25px;
   border: 1px solid #9390ff;
   background-color: #9390ff;
-  margin-left: 25px;
-  width: 40%;
-  margin-top: 40px;
-  margin-bottom: 60px;
+  margin-bottom: 40px;
   align-self: left;
+  margin-right: 780px;
+  margin-top: 20px;
```

Comment:
Kindly update your button padding to look like the one in the design 😉.

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199632535-bf47f6c0-91cb-4435-aaee-b1adf8a74166.png)  | ![image](https://user-images.githubusercontent.com/42799579/199632520-921178d0-ecfe-47dc-ab33-309fc83d5ea3.png)  |

Also, kindly add the corresponding button effects to your buttons: 
![image](https://user-images.githubusercontent.com/42799579/199632598-ab3884cc-3695-446c-9a5b-9eb2cf8909d8.png) 
You should implement the hover & pressed effects 😃. 

> Tip: you can always select an element on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) and a window will appear with all properties of that given element 😉:
![image](https://user-images.githubusercontent.com/42799579/199632717-cc1946fc-9fb1-4b08-ad5c-042fb9314a63.png)

Reply:
so how can I take it from the Figma?
coz its a picture , I'm confused

Reply:
> so how can I take it from the Figma? coz its a picture , I'm confused

Hello @maherramadan78 , you can select any element on Figma and it will display the properties for you. For example, if you click on a button, a window will be shown:
![image](https://user-images.githubusercontent.com/42799579/199649048-75a1e794-da76-43a7-982d-c67cfecf37c4.png)

Here you can see the colors, the gap, the background and everything that you need to follow the design properly 😉.